### PR TITLE
Adapter Rendering, Editbox, Label and Audio.

### PR DESCRIPTION
**Pull request by yyycggg**
Re: #https://github.com/cocos/3d-tasks/issues/13998

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: platforms/minigame/platforms/taobao/wrapper/engine/AssetManager.js [hunk 1]
```diff
@@ -2,33 +2,37 @@ const parser = cc.assetManager.parser;
...
+        '.ico': doNothing,
+        '.tiff': doNothing,
+        '.image': doNothing,
+        '.webp': doNothing,
     });
 }
 
-function loadImage(url, options, onComplete) {
+downloader.register({
+    // Note: Audio can't load from cached downloaded file onTaoBao
```
Comment (minggo) [inline platforms/minigame/platforms/taobao/wrapper/engine/AssetManager.js]: Please add todo.

File: platforms/minigame/platforms/taobao/wrapper/engine/AssetManager.js [hunk 2]
```diff
@@ -2,33 +2,37 @@ const parser = cc.assetManager.parser;
...
-        '.gif': downloadImage,
-        '.ico': downloadImage,
-        '.tiff': downloadImage,
-        '.image': downloadImage,
-        '.webp': downloadImage,
-        '.pvr': downloadImage,
-        '.pkm': downloadImage,
-        '.astc': downloadImage,
+        // NOTE: Image can't load from cached downloaded file on Taobao
+        '.png': doNothing,
```
Comment (minggo) [inline platforms/minigame/platforms/taobao/wrapper/engine/AssetManager.js:12]: Please add TODO in comments.

**Review by PPpro — APPROVED** [review]

File: platforms/minigame/platforms/taobao/wrapper/engine/AssetManager.js (hunk 1 above)
Comment (yyycggg) [reply platforms/minigame/platforms/taobao/wrapper/engine/AssetManager.js]: OK

File: platforms/minigame/platforms/taobao/wrapper/engine/AssetManager.js (hunk 2 above)
Comment (yyycggg) [reply platforms/minigame/platforms/taobao/wrapper/engine/AssetManager.js:12]: OK

**Review by minggo — APPROVED** [review]

**Pull request merged**